cli/cosipbftcontroller/mod.go

```diff
@@ -37,6 +37,8 @@ import (
 )
 
 const privateKeyFile = "private.key"
+const INJECTOR_ERROR = "injector: %v"
```

Comment:
Have a look at `contract/evoting.go` for a nice example of constant declaration. We also use the camel case convention for constants.